src/js/defaults.js

```diff
@@ -81,8 +81,8 @@ define('summernote/defaults', function () {
         },
         link: {
           link: 'link',
-          unlink: 'unlink',
-          edit: 'edit'
+          unlink: 'chain-broken',
+          edit: 'fa-pencil-square-o'
```

Comment:
I thought `fa-` must be removed. Because we have already iconPrefix.

https://github.com/summernote/summernote/blob/develop/src/js/defaults.js#L58